<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>app_1</class>
 <widget class="QWidget" name="app_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>436</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Fixedsys</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Manager Password</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/Clowns_Password.ico</normaloff>../img/Clowns_Password.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="user_habilitar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/esconder.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>saved passwords</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_app">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;
</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="salvar_nova_senha">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;
</string>
   </property>
   <property name="text">
    <string>save new password</string>
   </property>
  </widget>
  <widget class="QComboBox" name="nao_listado">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_leberar_acesso">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_leberar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="acesso_login">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="acesso_password">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="autenticar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;
</string>
   </property>
   <property name="text">
    <string>authenticate</string>
   </property>
  </widget>
  <widget class="QLabel" name="noti_liberar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Fixedsys&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/Clowns_Password_png.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
